_tests_/Ship.test.js

```diff
@@ -0,0 +1,64 @@
+/*globals describe it expect*/
```

Comment:
I dont think you need to have this here, the tests run fine without it being included so it would be safe to delete it